src/runtime/disco/threaded_session.cc

```diff
@@ -146,13 +152,17 @@ class DiscoThreadedMessageQueue : public dmlc::Stream {
       this->Read<int64_t>(&dref->reg_id);
       dref->session = Session{nullptr};
       result = ObjectRef(std::move(dref));
+    } else if (type_index == TypeIndex::kRuntimeString) {
+      uint64_t size = 0;
```

Comment:
To make things consistent, it is helpful to pass in string length, instead of len+1(that includes termination string)

Reply:
I'm not sure if my understanding is correct here. In `WriteObj`, we write the string's actual length to dmlc::Stream, and in `ReadObj` here, we create an `std::string` of length `len + 1`, and update its first `len` characters. I suppose in either case, the terminal character is not included.

Reply:
i misread the code. no questions for now